Sources/Testing/CMakeLists.txt

```diff
@@ -85,10 +85,12 @@ add_library(Testing
   Traits/TimeLimitTrait.swift
   Traits/Trait.swift)
 target_link_libraries(Testing PRIVATE
-  TestingInternals)
+  _TestingInternals)
 add_dependencies(Testing
   TestingMacros)
 target_compile_definitions(Testing PRIVATE
   SWT_BUILDING_WITH_CMAKE)
 target_compile_options(Testing PUBLIC
   -load-plugin-executable "${TestingMacrosPath}")
+target_compile_options(Testing PRIVATE
```

Comment:
I believe @bnbarham's advice was to apply this option to _TestingInternals as well.

Reply:
It's a cpp module, so I'm not sure how that would work?

Reply:
I'm assuming we build this module with `clang`?

Reply:
I'd have to defer to him!

Reply:
But yes, clang builds this code.

Reply:
OK.  Well, I tried it and as I suspected:
```
clang: error: unknown argument: '-enable-library-evolution'
```

I'm not sure what the equivalent would be.  @bnbarham?  I'm assuming it was assumed to be a Swift module?

Reply:
I don't know if I did or did not say this, but all I intended to say was "all swift modules", as in my other comment.

Reply:
It's only available on Swift. It needs to be wrapped in a generator expression. `$<$<COMPILE_LANGUAGE:Swift>:-enable-library-evolution>`

Reply:
`_TestingInternals` is all C++, all the time. That's the one that's causing some confusion.

Reply:
The purpose of all this is to try to ensure that `_TestingInternals` is not accessible outside `Testing`, to the best of our abilities. If `-enable-library-evolution` on our Swift target (`Testing`) helps, great. If not, then I am unsure how to proceed.

Reply:
I think at this point we've done everything we can.  We've made it pretty clear to consumers they're not meant to be using this directly.

Reply:
That's fair enough.